src/cmds/clean.c

```diff
@@ -469,6 +469,13 @@ enum swupd_code clean_statedir(bool dry_run, bool all)
 	path = statedir_get_manifest_root_dir();
 	ret = clean_staged_manifests(path, dry_run, all);
 	FREE(path);
+	if (ret != SWUPD_OK) {
```

Comment:
Isn't this supposed to be
`if (ret == SWUPD_OK)`?

If the provided "path" doesn't exist is going to return `SWUPD_COULDNT_LIST_DIR` and that means we need to carry on and try to remove from the old location. Am I missing something?

Reply:
We need to always clean both. I just followed the standard on clean to fail on first error.
this shouldn't fail for that reason because the path is created on statedir setup.

Reply:
ahh I see... you are right. So for the moment we will be cleaning both and eventually we'll remove this last check. Got it.